test/functional/sync_plan/info_test.rb

```diff
@@ -0,0 +1,54 @@
+require_relative '../test_helper'
+require 'hammer_cli_katello/sync_plan'
+require_relative 'sync_plan_helpers'
+require_relative '../organization/organization_helpers'
+module HammerCLIKatello
+  describe SyncPlan::InfoCommand do
+    include OrganizationHelpers
+    include SyncPlanHelpers
+
+    it 'by ID' do
+      id = 1
+      api_expects(:sync_plans, :show) do |p|
+        p['id'] == id
+      end
+      arguments = "sync-plan info --id #{id}"
+      run_cmd(arguments.split)
```

Comment:
Why use the `arguments` variable? Why not just `run_command(%W(sync-plan info --id #{id}))`?